hyperSpec/vignettes/plotting.Rmd

```diff
@@ -152,22 +152,21 @@ Besides, package **tripack**[`r cite_pkg("tripack")`] is mentioned, but not used
 
 # Preliminary Calculations {-}
 
-For some plots of the `chondro`{.r} dataset, the pre-processed spectra and their cluster averages $\pm$ one standard deviation are more suitable:
+For some plots of the `faux_cell`{.r} dataset, the pre-processed spectra and their cluster averages $\pm$ one standard deviation are more suitable:
 
-```{r preproc-chondro}
-chondro <- hyperSpec::chondro
+```{r preproc-faux-cell}
+set.seed(1)
+faux_cell <- generate_faux_cell()
 
-chondro.preproc <- chondro - spc.fit.poly.below(chondro)
-chondro.preproc <- chondro.preproc / rowMeans(chondro)
-chondro.preproc <- chondro.preproc - quantile(chondro.preproc, 0.05)
+faux_cell_preproc <- faux_cell - spc.fit.poly.below(faux_cell)
+faux_cell_preproc <- faux_cell_preproc / rowMeans(faux_cell)
```

Comment:
This pre-processing step may not be needed with `faux_cell`

Reply:
Baseline correction is a necessary step. 
The others (e.g., quantile correction) are not. Please, look at figures 11, 12, 13 in the current version of [`hyperSpec.ggplot2` vignette](https://r-hyperspec.github.io/hySpc.ggplot2/articles/hySpc.ggplot2.html#qplotmixmap). They illustrate the value of several pre-processing steps.

Reply:
Should I fix this section? (I.e., should I reduce the number of pre-processing steps?